src/main/java/com/assure/movie/model/repository/MemberRepository.java

```diff
@@ -2,11 +2,15 @@
 
 import com.assure.movie.model.domain.Member;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 /**
  * @author Israel Yasis
  */
 @Repository
```

Comment:
Do we need to add this annotation and why?

Reply:
the @Repository annotation is not necessary because Spring Data Scan and detect the interfaces which are extending from JpaRepository and then creates an implementation per every interface and after Spring Data take those implementations and  added them to application context as beans, everything happen during runtime, So thats the reason that those Interfaces can work without @Repository annotation.

Reply:
Yep, that is right!